src/helpers/levels.ts

```diff
@@ -48,25 +48,25 @@ export function computeLevelsQuote(
 
   for (let i = 1; i < levels.length; i++) {
     const nextLevel = levels[i]!;
-    const nextLevelDepth = nextLevel.level.minus(levels[i - 1]!.level);
+    const nextLevelDepth = nextLevel.q.minus(levels[i - 1]!.q);
```

Comment:
Note that v3 levels are not cumulative, but v2 levels are, so this logic probably needs to be updated.

Reply:
I updated this code. Please take another look, @sameer-hashflow.